### PR TITLE
added find by strategy

### PR DESCRIPTION
**Pull request by NadiMarkava**
added find by image strategy

### CODE REVIEW

File: src/main/java/common/pages/LoginPageBase.java [hunk 1]
```diff
@@ -16,4 +16,6 @@ public LoginPageBase(WebDriver driver) {
     public abstract ProductsPageBase loginAsUser(User user);
 
     public abstract String getErrorMessage();
+
+    public abstract boolean isImagePresent();
```
Comment (ankochergin) [inline src/main/java/common/pages/LoginPageBase.java]: is think method used anywhere? please add test for it

Comment (ankochergin) [inline src/main/java/common/pages/LoginPageBase.java]:
```suggestion
    public abstract boolean isRobotImagePresent();
```

File: src/main/java/common/pages/DrawingPageBase.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import org.openqa.selenium.WebDriver;
+
+public abstract class DrawingPageBase extends SwagLabsAbstractPageBase implements IMobileUtils {
+
+    public DrawingPageBase(WebDriver driver) {
+        super(driver);
+    }
+
+    public abstract void drawPicture();
```
Comment (ankochergin) [inline src/main/java/common/pages/DrawingPageBase.java]:
```suggestion
    public abstract void drawLine();
```

File: src/main/java/android/pages/DrawingPage.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @Override
+    public void drawPicture() {
+        int startX = signature.getLocation().getX() + signature.getSize().getWidth() / 10;
+        int startY = signature.getLocation().getY() + signature.getSize().getHeight() / 10;
+        int endX = startX * 7;
+        swipe(startX, startY, endX, startY, 1000);
+    }
+
+    @Override
+    public boolean isDrawingPresent() {
```
Comment (ankochergin) [inline src/main/java/android/pages/DrawingPage.java]:
```suggestion
    public boolean isDrawnLinePresent() {
```

Comment (NadiMarkava) [reply src/main/java/android/pages/DrawingPage.java]: done

File: src/main/java/common/pages/DrawingPageBase.java (hunk 2 above)
Comment (NadiMarkava) [reply src/main/java/common/pages/DrawingPageBase.java]: done

File: src/main/java/common/pages/LoginPageBase.java (hunk 1 above)
Comment (NadiMarkava) [reply src/main/java/common/pages/LoginPageBase.java]: done

Comment (NadiMarkava) [reply src/main/java/common/pages/LoginPageBase.java]: done

File: src/test/java/android/MobileSampleTest.java [hunk 4]
```diff
@@ -31,6 +31,7 @@ public void verifyUserCanLogOut() {
         Assert.assertTrue(productsPage.isPageOpened(), "Products page isn't opened");
         LoginPageBase loginPage = (LoginPageBase) getNavUtil().clickNavMenuLink(NavMenu.LOG_OUT);
         Assert.assertTrue(loginPage.isPageOpened(), "Login page isn't opened");
+        Assert.assertTrue(loginPage.isRobotImagePresent(), "Image isn't present");
```
Comment (ankochergin) [inline src/test/java/android/MobileSampleTest.java]: I think it shouldn't be part of this test, better to create a separate test for it. This test is to verify only logout

**Pull request merged**